subiquitycore/i18n.py

```diff
@@ -29,9 +29,9 @@
 
 
 def switch_language(code='en_US'):
-    if code != 'en_US' and 'FAKE_TRANSLATE' in os.environ:
+    if os.environ.get('FAKE_TRANSLATE', "0") != "0":
         def my_gettext(message):
-            return "_(%s)" % message
+            return message
```

Comment:
Why did you change this? It's useful to see which strings are not being translated (and also which bits of the UI make assumptions about string length)

Reply:
a unit test failed without changing that. let me see.

Reply:
unit tests fall over baly if i do not change this.
  [see here](http://paste.ubuntu.com/p/YQgz2YVHWD/)

Reply:
Oh right. Can we key off the value of FAKE_TRANSLATE then? FAKE_TRANSLATE=mangle keeps the behaviour from master, or something like that?

Reply:
FAKE_TRANSLATE=mangle will take path that was in place before.